src/Main.java

```diff
@@ -0,0 +1,63 @@
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        ArrayList<MatchStats> listOfMatch = new ArrayList<>();
+        ArrayList<String> namesOfTeam = new ArrayList<>();
+        ArrayList<Team> listOfTeam = new ArrayList<>();
+        Scanner inputName = new Scanner(System.in);
+
+        System.out.print("Enter file name to read: ");
+        String fileName = inputName.nextLine();
+        String filePath = "DataSource/" + fileName;
+
+        try (Scanner scanner = new Scanner(Paths.get(filePath))) {
+            while (scanner.hasNextLine()) {
+                String[] currentLine = scanner.nextLine().split(",");
+                int scoreOne = Integer.parseInt(currentLine[2]);
+                int scoreTwo = Integer.parseInt(currentLine[3]);
+
+                listOfMatch.add(new MatchStats (currentLine[0], currentLine[1], scoreOne, scoreTwo));
+                currentLine = null;
+            }
+        } catch (Exception e) {
+            System.out.println("Reading the file " + fileName + " failed.");
+        }
+
+        // Creating list of teams
+        for (MatchStats match : listOfMatch) {
+            if (!(namesOfTeam.contains(match.getTeamOne()))) {
+                namesOfTeam.add(match.getTeamOne());
+            }
+            if (!(namesOfTeam.contains(match.getTeamTwo()))) {
+                namesOfTeam.add(match.getTeamTwo());
+            }
+        }
+
+        // Creating each team as object
+        for (String team : namesOfTeam) {
```

Comment:
Myslim si, ze je zbytocne udrzovat zoznamy namesOfTeam a listOfTeam, ja by som to videl takto:
- vytvoris zoznam listOfMatch
- poziadas o user input teamName
- zacnes iterovat nad listOfMatch a zbieras statistiky o muzstve s danym menom. Na to by mohla sluzit treba trieda MatchAnalyzer, ktory vrati objekt typu Team - uz zcela zpracovane. V pripade, ze taky team neexistuje, vratis null, alebo vyhodis vynimku.

Reply:
Pokud tym neexistuje sem osetril pomoci IF, nevim jestli je lepsi pouzit TRY v tomto pripade?
Dodelal sem tam dve tridy "TeamAnalyzer" a "Match Analyzer", ale zda se mi to cely nejaky prekombinovany.
Nemuzu projit na to, jak zkombinovat ty funkce trid "TeamAnalyzer" a "Match Analyzer", abych v "Main" volal uz napriklad jenom
"TeamAnalyzer" pro zobrazeni informaci k pozadovanemu tymu.

Reply:
`If` je lepsia moznost v tomto pripade, vyklad bude nasledovat, az berieme exceptiony na hodine :D